tests/models/files/test_details.py

```diff
@@ -0,0 +1 @@
+# No tests here...yet.
```

Comment:
Don't think there are any appropriate unit tests for this module, since right now it's basically just a big dictionary.